src/components/ha-sidebar.ts

```diff
@@ -292,106 +295,92 @@ class HaSidebar extends LitElement {
                       ? "mdi:view-dashboard"
                       : panel.icon}
                   ></ha-icon>
-                  <span class="item-text"
-                    >${panel.url_path === this.hass.defaultPanel
-                      ? hass.localize("panel.states")
-                      : hass.localize(`panel.${panel.title}`) ||
-                        panel.title}</span
-                  >
+                  ${panel.url_path === this.hass.defaultPanel
+                    ? hass.localize("panel.states")
+                    : hass.localize(`panel.${panel.title}`) || panel.title}
                   <mwc-icon-button class="show-panel">
                     <ha-svg-icon .path=${mdiPlus}></ha-svg-icon>
                   </mwc-icon-button>
-                </paper-icon-item>`;
+                </mwc-list-item>`;
               })}
-              <div class="spacer" disabled></div>
+              <mwc-list-item class="spacer" disabled></mwc-list-item>>
             `
           : ""}
         ${this._renderPanels(afterSpacer)}
         ${this._externalConfig && this._externalConfig.hasSettingsScreen
           ? html`
-              <a
-                aria-role="option"
-                aria-label=${hass.localize(
-                  "ui.sidebar.external_app_configuration"
-                )}
-                href="#external-app-configuration"
-                tabindex="-1"
-                @click=${this._handleExternalAppConfiguration}
-                @mouseenter=${this._itemMouseEnter}
-                @mouseleave=${this._itemMouseLeave}
-              >
-                <paper-icon-item>
+              <mwc-list-item>
+                <a
```

Comment:
The anchor should be outside of the item, you can no longer right click and open in a new tab now. Also keyboard navigation doesn't work now.